tests/D2L.CodeStyle.Analyzers.Test/SpecRunner.cs

```diff
@@ -272,7 +272,7 @@ ImmutableArray<string> arguments
 
 				return new DiagnosticExpectation(
 					name: name,
-					arguments: arguments.Split( ',' ).ToImmutableArray()
+					arguments: arguments.Split( ',' ).Select( arg => arg.Trim() ).ToImmutableArray()

```

Comment:
This is the thing that runs the spec tests. Instead of having to say `/* SomeDiag(arg1,arg2,arg3) */ blah /**/` I wanted to be able to say `/* SomeDiag(arg1, arg2, arg3) */ blah /**/`